### PR TITLE
fix(popover): Allow nesting of popovers

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #4009

## Summary

fix(popover): Allow nesting of popovers. #4009

Break words instead of hiding overflows of content.

### CODE REVIEW

**Comment by driskull** [comment]
@asangma @jcfranco can you review this one?

File: src/demos/popover.html [hunk 1]
```diff
@@ -103,13 +103,38 @@ <h1 style="margin: 0 auto; text-align: center">Popover</h1>
...
+        <style>
+          .info {
+            padding: 10px;
+          }
+
+          .header {
+            font-weight: bold;
+          }
+        </style>
+        <div class="info">
```
Comment (jcfranco) [inline src/demos/popover.html:122]: Can this be converted into a nested-popover story?

**Review by jcfranco — APPROVED** [review]
Code LGTM! I'll defer to @asangma on the CSS.

File: src/demos/popover.html (hunk 1 above)
Comment (driskull) [reply src/demos/popover.html:122]: Yeah good idea. I'll add one.

File: src/components/popover/popover.scss [hunk 2]
```diff
@@ -67,11 +66,16 @@
   flex-col
   flex-nowrap
   self-center
-  overflow-hidden
+  word-break
   h-full
   w-full;
 }
 
+.close-button {
```
Comment (driskull) [inline src/components/popover/popover.scss:74]: @asangma these were added so that the close button doesn't mess with the rounded corners of the popover.

**Comment by driskull** [comment]
Story added, Waiting on CSS feedbackz

File: src/components/popover/popover.scss (hunk 2 above)
Comment (asangma) [reply src/components/popover/popover.scss:74]: 👀

Comment (asangma) [reply src/components/popover/popover.scss:74]: We might be able to remove those margins if we added a div around Action.close-button, gave that wrapper appropriate border-radius (for RTL and LTR) and `overflow: hidden`.

The wrapper would need to be flex and probably `flex: 0 0 auto`

Too much?

image

Comment (driskull) [reply src/components/popover/popover.scss:74]: I'll give it a shot. If I can't get it to work can I hand it off to you?

**Review by asangma — APPROVED** [review]
Beaut!

**Comment by asangma** [comment]
re: Screener
I dunno what's up but it looks like it's not showing the rounded corners on the close buttons.

**Comment by driskull** [comment]
> I dunno what's up but it looks like it's not showing the rounded corners on the close buttons.

Yeah, this is because its not screener 97 so it doesn't support those CSS logical properties. It should be good in local tests.

@jcfranco @eriklharper more reason to get screener bumped.

**Comment by asangma** [comment]
Ah...got it.

**Pull request merged**